en/appendices/3-5-migration-guide.rst

```diff
@@ -180,6 +180,7 @@ Routing
   match.
 * ``Cake\Routing\RouteBuilder`` now has methods to create routes for
   specific HTTP methods. e.g ``get()`` and ``post()``.
+* ``Cake\Routing\RouteBuilder::loadPlugin()`` was added.
```

Comment:
Fine, but https://github.com/cakephp/cakephp/pull/10711/files#diff-7997caf44fa801d62e790043f405bd34R332 needs to be removed then.